values.yaml

```
@@ -1276,7 +1276,10 @@ nginx:
     #   setHeaders:
     #     X-Scope-OrgID: $remote_user
     basicAuthSecretName: ""
-
+    # -- (optional) Including the valid parameter to the `resolver` directive to re-resolve names every `dnsValid` seconds/minutes
+    dnsValid: ""
```

Since this is broken now, I would be happy to see a default that behaves nicely.
```suggestion
    dnsValid: "15s"
```

---

good idea

---

Yeah great idea. Anyone wanna change the option name? I see in the other PR they have dnsTTL. 

By the way, after I re-tested the whole case here, it looks like the `valid=15s` is all we need to solve the problem (no headless service change required).

---

yeah I like dnsTTL more